Resources/Prototypes/Reagents/narcotics.yml

```diff
@@ -314,6 +314,16 @@
   physicalDesc: reagent-physical-desc-acrid
   flavor: bitter
   color: "#215263"
+  metabolisms:
+    Narcotic:
+      effects:
+      - !type:MovespeedModifier
+        walkSpeedModifier: 0.65
+        sprintSpeedModifier: 0.65
+      - !type:HealthChange
+        damage:
+          types:
+            Cellular: 4 # TODO: Make this deal brain damage instead of genetic damage once brain damage is added
```

Comment:
God no.

We already have a growing section of players who are dumping spaceacillin into bodies to make them unclonable, and this just makes that strategy even more viable. Getting permanently taken out of the round because somebody injected a hypospray of this into you with no possible counterplay (it even slows you down so you can’t run away!) is horrible design. I don’t care if it’s *technically* faithful to how it worked in ss13, a dedicated permakill chem is not what the game needs, no matter how challenging the recipe is.

Reply:
Replaced cellular damage with poison as suggested.